internal/kafka/command_topic_produce_test.go

```diff
@@ -1,8 +1,10 @@
 package kafka
 
 import (
+	"fmt"
 	"testing"
 
+	ckafka "github.com/confluentinc/confluent-kafka-go/kafka"
```

Comment:
This import belongs at the bottom, separated from the third-party imports by a newline.

Reply:
Does it belong all the way to the bottom, I saw in other files it's in the middle but separate by newline. Let me know.

Reply:
It should be below third-party imports, but above CLI-specific imports.